airbyte-integrations/connectors/source-google-ads/source_google_ads/google_ads.py

```diff
@@ -212,10 +212,21 @@ def get_field_value(field_value: GoogleAdsRow, field: str, schema_type: Mapping[
             field_value = str(field_value)
 
         return field_value
+    
+    @staticmethod
+    def convert_array(field_value: List[str]) -> List[Dict[str, any]]:
```

Comment:
I am tending to ask you to revert changes related to "string convertion to dict".
Reasons:

1. Current code does not properly convert strings with nested structutures like:
	ad_group_ad_legacy
	
	"ad_group_ad.ad.responsive_search_ad.descriptions": [ "text: \"Behind The Scenes: Testing The Airbyte Maintainer Program\"\nasset_performance_label: PENDING\npolicy_summary_info {\n  review_status: REVIEWED\n  approval_status: APPROVED\n}\n"]
  
    so output should be:
	
	"ad_group_ad.ad.responsive_search_ad.descriptions": [
		{
			text: Consolidate your data in your data warehouses, lakes and databases
			asset_performance_label: PENDING
			policy_summary_info {
				review_status: REVIEWED
				approval_status: APPROVED
			}
		}
	]

2. Schema changes: some type of fields was chnaged from "list of strings" to "list of objects" without specifying fields for "objects". It will lead to empty values on destination side. Instead, it is better to have a string with full content
	
	IN MASTER:
    "ad_group_ad.ad.responsive_search_ad.descriptions": {
      "type": [
        "null",
        "array"
      ],
      "items": {
        "type": "string"
      }
    },
	
	IN PR:
    "ad_group_ad.ad.responsive_search_ad.descriptions": {
      "type": [
        "null",
        "array"
      ],
      "items": {
        "type": "object"
      }
    },
	

So we should better revert changed related to "string converion" in python code and in schema.

Reply:
Hello @midavadim i update my function to include your recommandation 1. you're right !
For the second point, I know it's a breaking change, but I put object because it depends on the account and the field we have. If there is no data, it just put **"ad_group_ad.ad.responsive_search_ad.descriptions": []**

For us, it was really a big issue to extract the data, and this way of doing things helped us a lot. 

Need to reopen the PR to see the changes